lib/zoom/actions.rb

```diff
@@ -26,31 +26,40 @@ def self.determine_request_options(client, oauth)
       end
     end
 
-    def self.make_request(client, method, parsed_path, params, request_options)
+    def self.make_request(args)
+      client, method, parsed_path, params, request_options, oauth =
+        args.values_at :client, :method, :parsed_path, :params, :request_options, :oauth
```

Comment:
I'm not sure how this is beneficial over the old list of arguments in the function definition. It seems like a refactor that may be outside the scope of this change.

Reply:
@joelmichael I just considered, that more than five arguments is too much, that's why I enclosed it into just one hash argument.